### ISSUE TITLE
[BUG] MigratorServiceLoader does not work with GraalVM native-image

### ISSUE THREAD

**Issue by michelu89** [eclipse-esmf/esmf-sdk#288]
``**Describe the bug**
When creating a NATIVE IMAGE with GraalVM, the MigratorService.class will not be instantiated. 

**Where**
sds-aspect-model-version-migrator version 2.1.1.
Class ->  io.openmanufacturing.sds.aspectmodel.versionupdate.MigratorServiceLoader

**Additional context**
The problem could be that the MigrationService is scanned at runtime in the classPath and it is not present in the native-image. (see below)

```
private void loadMigratorService() {
      try ( final ScanResult scanResult = new ClassGraph().enableAllInfo().whitelistPackages(
            MigratorFactory.class.getPackageName() ).scan() ) {
         final ClassInfoList migratorFactoryClasses = scanResult
               .getClassesImplementing( MigratorFactory.class.getName() );

         migratorService = migratorFactoryClasses
               .loadClasses( MigratorFactory.class ).stream().map( this::createMigratorFactory )
               .map( MigratorService::new ).findAny().orElse( new MigratorService() );
      }
   }
```

Stacktrace: [stacktrace.txt](https://github.com/eclipse-esmf/esmf-sdk/files/10507882/stacktrace.txt)

**Comment by atextor** [comment]
@michelu89 For now, this can be adressed using a GraalVM substitution. In the not-yet merged feature branch for #140, such a substitution is already implemented for the native-image build of the bamm-cli:
https://github.com/bci-oss/sds-sdk/blob/140-native-binaries/tools/bamm-cli/src/main/java/io/openmanufacturing/sds/substitution/Target_io_openmanufacturing_sds_aspectmodel_versionupdate_MigratorServiceLoader.java
Please check if this works for you.

**Comment by michelu89** [comment]
This works fine for me. Thank you!
